libc-test/build.rs

```diff
@@ -302,6 +302,7 @@ fn main() {
             // Fixup a few types on windows that don't actually exist.
             "time64_t" if windows => "__time64_t".to_string(),
             "ssize_t" if windows => "SSIZE_T".to_string(),
+            "pid_t" if windows => "pid_t".to_string(),
```

Comment:
I think this may be caught by the clause below, right?

Reply:
Drat. I totally missed that. Too right.

Reply:
i have been reading too much lisp code. I read the "t if" as "true if" :-)

Reply:
Heh no worries!

Reply:
I have a fresh Win10 VM running so I should be able to test this locally finally.